src/mpid/ch4/netmod/ofi/ofi_pre.h

```diff
@@ -45,6 +45,7 @@ typedef struct {
 typedef struct {
     /* support for connection */
     int conn_id;
+    int committed;
```

Comment:
Not sure this is needed. Re-evaluate after addressing the previous comments.

Reply:
Not needed now